libraries/bot-applicationinsights/src/test/java/com/microsoft/bot/applicationinsights/TelemetryInitializerTests.java

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.bot.applicationinsights;
+
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import com.microsoft.applicationinsights.TelemetryClient;
+import com.microsoft.bot.applicationinsights.core.TelemetryInitializerMiddleware;
+import com.microsoft.bot.builder.BotTelemetryClient;
+import com.microsoft.bot.builder.TelemetryLoggerMiddleware;
+import com.microsoft.bot.builder.adapters.TestAdapter;
+import com.microsoft.bot.builder.adapters.TestFlow;
+import com.microsoft.bot.schema.Activity;
+import com.microsoft.bot.schema.ActivityTypes;
+
+public class TelemetryInitializerTests {
+
+    @Test
+    public void telemetryInitializerStoresActivity() {
+        TelemetryClient telemetryClient = new TelemetryClient();
+
+        /**
+         * TODO: Check Typescript implementation
+         */
+    }
+
+    @Test
+    public void telemetryInitializerMiddlewareLogActivitiesWhenEnabled() {
+    	TelemetryClient mockTelemetryClient = Mockito.mock(TelemetryClient.class);
+
+        TelemetryLoggerMiddleware telemetryLoggerMiddleware = new TelemetryLoggerMiddleware((BotTelemetryClient) mockTelemetryClient, false);
```

Comment:
```suggestion
        TelemetryLoggerMiddleware telemetryLoggerMiddleware = new TelemetryLoggerMiddleware(mockTelemetryClient, false);
```

Changing the mock of above, this should be removed